index.html

```diff
@@ -142,18 +142,43 @@ <h1>Adding a Specification Entry</h1>
   specifications to address a variety of real world use cases. Examples of
   extensions include, but are not limited to, new types of Verifiable Credentials
   and ways of expressing credential status, schema validation, evidence,
-  refreshing, terms of use, and cryptographic suites for securing credentials. In
+  refreshing, terms of use, and cryptographic suites for securing credentials.
+     </p>
+     <p> In
   order to add a new specification to this directory, an implementer submits a
   modification request for this directory, as a pull request on the repository
-  where this directory is hosted, where the modification request adheres to the
-  following policies:
-      </p>
-
+  where this directory is hosted.
+  </p>
+
+  Here is an example:
+
+  <p>
+  <pre class="example">
+{
+  // These fields are required
+  "name": "Example VC",
+  "summary": "Used to demonstrate examples for Verifiable Credentials.",
+  "specification": "https://example.github.io/example-spec/",
+  // categories include: vc, credentialStatus, credentialSchema,
+  //                     refreshService, termsOfUse, evidence, and proof
+  "category": "vc",
+  "maintainerEmail": "maintainer@community.example",
+  // These fields are optional
+  "maintainerName": "Example Community Group",
+  "maintainerWebsite": "https://example.github.io/",
+  // RDF vocabularies in yml2vocab format
+  "vocabulary": ["https://example.github.io/vocabularies/example.yml"]
```

Comment:
Should we have comments in the example? I get that it helps explain the example; it's also invalid JSON.

I'm fine either way, just raising it as a concern.

Reply:
I removed the comments